src/ScheduleFunctions.cpp

```diff
@@ -2269,6 +2269,7 @@ bool validate_schedule(Function f, const Stmt &s, const Target &target, bool is_
 
     std::ostringstream err;
 
+    /*
```

Comment:
If this is meant to left here in commented-out form, it is essential you add a comment explaining why. (Otherwise, just delete it.)

Reply:
Oops, deleted.